### PR TITLE
Remove user accounts for the agency logs users from transition-logs-1

### PR DESCRIPTION
**Pull request by issyl0**
- The transition-logs-1 machine was used during Transition to enable us
  to receive and process webserver logs from agencies, to generate
  mappings and hits data for their sites pre-transition. Transition is
  done, in this case, and we haven't received or processed any logs
  since November 2014. Therefore, in the interest of tidying up, remove
  these users.
- This commit does _not_ touch CDN logs - they're still being processed
  on this machine and need to be for up-to-date hits data
  post-transition for sites.

Tested this in dev, and it no longer creates the users, and their directories don't exist.

### CODE REVIEW

**Comment by jennyd** [comment]
Is this enough to remove the existing agency users, or do we also need to ensure that they're absent?

File: modules/ci_environment/manifests/transition_logs/remote_users.pp [hunk 1]
```diff
@@ -1,45 +0,0 @@
...
-
-  # Generate random cron times seeded with fqdn and username
-  $hour = fqdn_rand(8, $name)
-  $min  = fqdn_rand(59, $name)
-
-  # get backup target
-  # set a default of scp to localhost as the vagrant user.
-  # if testing stuff you will need to generate ssh keys
-  # and then add the public key to vagrant's authorized_keys.
-  $backup_target = hiera('ci_environment::transition_logs::backup_target', false)
```
Comment (issyl0) [inline modules/ci_environment/manifests/transition_logs/remote_users.pp]: Deleting these lines negates the need for https://github.gds/gds/ci-deployment/pull/58.

**Comment by issyl0** [comment]
@jennyd Hrm. I don't really know how Puppet works. I'll try destroying and provisioning the machine on master and then reprovisioning with these changes, rather than provisioning for the first time on this branch.

**Comment by ajlanghorn** [comment]
@issyl0 Further to @jennyd's comment, this makes use of [gds-accounts](https://github.com/alphagov/puppet_gds-accounts), per the Puppetfile. That module allows you to purge users - see https://github.com/alphagov/puppet-gds_accounts/blob/master/manifests/init.pp#L15-L18.

**Comment by issyl0** [comment]
Closing this while I attempt to understand whether just removing all of the account lines was the right thing to do for already provisioned machines (rather than just the one I brought up for the first time in dev), and if not, what is the right thing to do.

**Comment by ajlanghorn** [comment]
@issyl0 If you'd like help, feel free to ask.

**Comment by issyl0** [comment]
Nope, you're right: provisioning on master and then reprovisioning on this branch didn't remove the users.

**Pull request closed without merging**